scripts/performance.ps1

```diff
@@ -94,41 +94,48 @@ function Run-Foreground-Executable($File, $Arguments) {
     return $p.StandardOutput.ReadToEnd()
 }
 
-$GitPath = Join-Path $RootDir "artifacts/PerfDataGit"
-
-function Clone-Data-Repo() {
-    # Redirect stderr to stdout for git.
-    $env:GIT_REDIRECT_STDERR = '2>&1'
-    git clone --single-branch --branch data/performance https://github.com/microsoft/msquic $GitPath
-    $currentLoc = Get-Location
-    Set-Location -Path $GitPath
-    git clean -d -x -f
-    git reset --hard
-    git pull
-    Set-Location -Path $currentLoc
+function Parse-Loopback-Results($Results) {
+    #Unused variable on purpose
+    $m = $Results -match "Total rate.*\(TX.*bytes @ (.*) kbps \|"
+    return $Matches[1]
 }
 
-function Get-Last-Result($Path) {
-    $FullLatestResult = Get-Item -Path $Path | Get-Content -Tail 1
-    $SplitLatestResult =  $FullLatestResult -split ','
-    $LatestResult = $SplitLatestResult[$SplitLatestResult.Length - 1].Trim()
+function Get-Latest-Test-Results($Platform, $Test) {
+    $Uri = "https://msquicperformanceresults.azurewebsites.net/performance/$Platform/$Test"
+    Write-Host $Uri
+    $LatestResult =  Invoke-RestMethod -Uri $Uri
+
+    Write-Host $LatestResult
+
     return $LatestResult
 }
 
-function Parse-Loopback-Results($Results) {
-    #Unused variable on purpose
-    $m = $Results -match "Total rate.*\(TX.*bytes @ (.*) kbps \|"
-    return $Matches[1]
+function Median-Test-Results($FullResults) {
+    $sorted = $FullResults | Sort-Object
+    return $sorted[[int](($sorted.Length - 1) / 2)]
 }
 
+class TestPublishResult {
+    [string]$PlatformName
+    [string]$TestName
+    [string]$CommitHash
+    [double[]]$IndividualRunResults
+}
+
+$currentLoc = Get-Location
+Set-Location -Path $RootDir
+$env:GIT_REDIRECT_STDERR = '2>&1'
+$CurrentCommitHash = git rev-parse HEAD
+Set-Location -Path $currentLoc
+
 function Run-Loopback-Test() {
     Write-Host "Running Loopback Test"
     $proc = Start-Background-Executable -File $PingClient -Arguments "-listen:* -selfsign:1 -peer_uni:1"
     Start-Sleep 4
 
     $allRunsResults = @()
 
-    1..10 | ForEach-Object {
+    1..2 | ForEach-Object {
```

Comment:
```suggestion
    1..10 | ForEach-Object {
```

Reply:
Done. I drop it to make testing quicker and then forget to bring it back up.